File: test_login.js
async function testLogin() {
    try {
        const response = await fetch('http://localhost:5000/api/auth/login', {
            method: 'POST',
            headers: {
                'Content-Type': 'application/json'
            },
            body: JSON.stringify({
                username: 'personel_alanya',
                password: '123'
            })
        });
        const data = await response.json();
        console.log("LOGIN SUCCESS:", data);
    } catch (err) {
        console.error("LOGIN ERROR:", err);
    }
}

testLogin();

File: test_login.py
import urllib.request
import json

req = urllib.request.Request('http://localhost:5000/api/auth/login', 
                             data=json.dumps({"username": "Enes_d", "password": "123"}).encode('utf-8'),
                             headers={'Content-Type': 'application/json'})
try:
    with urllib.request.urlopen(req) as response:
        print(response.read().decode())
except Exception as e:
    print(e)
